ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/manager/AbstractRetryingPeerTask.java

```diff
@@ -68,12 +65,16 @@ protected void executeTask() {
               if (error != null) {
                 handleTaskError(error);
               } else {
+                // If we get a partial success reset the retry counter.
+                if (peerResult.size() > 0) {
+                  retryCount = 0;
```

Comment:
You can remove `resetRetryCounter()` now.

Reply:
Github merge restored it.

Reply:
note to self: never use github web merge again.